docs/guides/gke_github.md

```diff
@@ -200,8 +214,9 @@ resource "kubernetes_secret" "main" {
   }
 
   data = {
-    known_hosts = local.known_hosts
-    password    = var.github_token
```

Comment:
This secret data was invalid.
If we want to use basic authentication, `username` and `password` are required.
If we want to use OpenSSH, `identity`, `identity.pub`, `known_hosts` are required.